rcb4/armh7interface.py

```diff
@@ -463,6 +463,66 @@ def servo_error(self, servo_ids=None):
             ServoStruct, slot_name='error_angle')[servo_ids]
         return error_angles
 
+    def adjust_angle_vector(self, servo_ids=None, error_threshold=None):
+        """Stop servo motor when joint error become large.
+
+        This function stops servo motor when the error between reference angle
+        vector and current angle vector exceeds error threshold.
+
+        Parameters
+        ----------
+        servo_ids : array_like
+            Array of error check target servo IDs. Each ID corresponds to a
+            specific servo.
+        Error threshold : int or float
+            Error threshold angle [deg].
+
+        Returns
+        -------
+        bool
+            Return True when adjustment occurs.
+        """
+        if servo_ids is None:
+            servo_ids = self.search_servo_ids()
+        # Ignore free servo
+        servo_ids = servo_ids[
+            self.reference_angle_vector(servo_ids=servo_ids) != 32768]
+        if len(servo_ids) == 0:
+            return
+
+        # Calculate error threshold[deg]
+        if error_threshold is None:
+            error_threshold = 5
+        if isinstance(error_threshold, (int, float)):
```

Comment:
```
if isinstance(error_threshold, numbers.Number):
    error_threshold = np.full(len(servo_ids), error_threshold, dtype=np.float32)
```
is better, because isinstance(error_threshold, (int, float)) don't return True when error_threshold is numpy.int32 and numpy.float32.

Reply:
Thank you, I pushed [`d5787ba` (#76)](https://github.com/iory/rcb4/pull/76/commits/d5787ba67d2f6d82848e81b44a4bd85c65562733)